packages/scene-composer/src/components/panels/scene-rule-components/SceneRuleTargetEditor.tsx

```diff
@@ -98,7 +98,7 @@ export const SceneRuleTargetEditor: React.FC<ISceneRuleTargetEditorProps> = ({
             chosenColor={chosenColor}
           />
           {isCustomStyle && (
-            <ColorPicker
+            <TagColorPicker
```

Comment:
We have used this component in other components as well. So keep a generic name as well something like `ColorSelectorCombo`